python/mxnet/foo/model/vision.py

```diff
@@ -1,37 +1,14 @@
 from __future__ import division
 
-import argparse, time
```

Comment:
OK. Then we just need to keep in mind that we should maintain this for future interface changes.